pandas/tests/test_window.py

```diff
@@ -54,6 +56,38 @@ def arithmetic_win_operators(request):
     return request.param
 
 
+@pytest.fixture(params=['right', 'left', 'both', 'neither'])
```

Comment:
some of this seems a good candidate for a pre-cursor PR to master (the fixtures)